packages/expo-glass-effect/src/isGlassEffectAPIAvailable.ios.ts

```diff
@@ -0,0 +1,19 @@
+import { requireNativeModule } from 'expo-modules-core';
+
+let IS_GLASS_EFFECT_API_AVAILABLE: boolean | undefined;
+
+/**
+ * Checks whether the Liquid Glass API is available at runtime on the device.
+ *
+ * This API was added because some iOS 26 beta versions do not have this API available, which can
```

Comment:
```suggestion
 * This method was added because some iOS 26 beta versions do not have this API available, which can
```

I think we should call out when its a prop/method/hook/component in general in a library for clarity.